crates/bevy_core_pipeline/src/lib.rs

```diff
@@ -35,7 +35,9 @@ use bevy_render::{
     RenderApp, RenderStage, RenderWorld,
 };
 
-/// Resource that configures the clear color
+/// The color that is used to clear the screen between frames.
```

Comment:
```suggestion
/// When used as a resource, sets the color that is used to clear the screen between frames.

```
The extra line between the "basic description" and "details" is a helpful Rust convention that makes doc strings easier to use in IDEs.